playground/component/src/lib.rs

```diff
@@ -0,0 +1,33 @@
+#[allow(warnings)]
+mod bindings;
+
+use bindings::Guest;
+
+struct Component;
+
+struct StringWriter(pub String);
+
+impl wasmprinter::Print for StringWriter {
+    fn write_str(&mut self, s: &str) -> std::io::Result<()> {
+        self.0.push_str(s);
+        Ok(())
+    }
+}
```

Comment:
If you're interested in getting particularly fancy here it might be neat to hook into the `start_*` methods and emit `<span>` entries to have highlighted colos on the web too. That's fine to do later though as a follow-up as well.

Reply:
[Done](https://bakkot-org.github.io/wasm-tools/print), following roughly the terminal colors used for the same parts.